R/Gargle-class.R

```diff
@@ -0,0 +1,213 @@
+#' Generate a Gargle token
+#'
+#' Constructor function for objects of class [`Gargle2.0`]. The `"email"` scope
+#' is always added if not already present. This is needed to retrieve the email
+#' address associated with the token. This is considered a "low value" scope and
+#' does not appear on the consent screen.
```

Comment:
I'd throw a link to https://developers.google.com/+/web/api/rest/oauth#email or the like in here? (Maybe I'm link-happy.)